data/project/ember/release.md

```diff
@@ -5,12 +5,12 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.18.0 # Manually update, see https://libraries.io/npm/ember-source throughout
-initialReleaseDate: 2020-04-14 # Manually update
-lastRelease: 3.18.1 # Manually update
-futureVersion: 3.18.2 # Manually update
+initialVersion: 3.20.0 # Manually update, see https://libraries.io/npm/ember-source throughout
+initialReleaseDate: 2020-07-29 # Manually update
```

Comment:
According to https://www.npmjs.com/package/ember-source, the initial release was July  17, I think?

Reply:
Oh you're correct it's off, but I think you mean July 13, right?